### DISCUSSION TITLE
H5在iOS微信上打开App Store

### DISCUSSION THREAD

**Discussion by giscus** [wtto00/wtto00.github.io#80] [Announcements]
# H5在iOS微信上打开App Store

在微信浏览器中，就`访问iOS App Store链接直接打开App Store`这一个功能，能不能做个判断，当用户从公众号菜单，以及二维码扫描进入的页面时，则直接访问iOS App Store链接，让微信提示用户是否前往App Store；当用户直接点击url链接进入的页面，则弹出遮罩，提示用户前往浏览器打开。

http://localhost:4321/posts/ios-open-app-store-in-wechat/

**Comment by wtto00** [comment]
测试一下
